_automating-configurations/security.md

```diff
@@ -4,31 +4,31 @@ title: Workflow security
 nav_order: 25
```

Comment:
Does the navigation looks good @kolchfa-aws ?

Reply:
I moved this topic down so it's the last in the left nav.